### PR TITLE
Add DesktopViewer to VmConsoles

### PR DESCRIPTION
**Pull request by mareklibra**
With this patch the DesktopViewer is added among other console types.

TODO:
- [x] have pf-react #989 merged, upgrade dependency version
- [x] maybe a follow-up: pass console details as an object instead of getter function
- [x] render only if either VM's VNC or RDP service is exposed (port, route)

### CODE REVIEW

File: sass/components/_VmConsoles.scss [hunk 1]
```diff
@@ -2,3 +2,7 @@
   padding-bottom: 12px !important;
   margin-top: -14px !important;
 }
+
+.vm-consoles-no-rdp .expand-collapse-pf-link-container {
```
Comment (rawagner) [inline sass/components/_VmConsoles.scss]: please use BEM with kubevirt prefix

File: src/components/VmConsoles/VmConsoles.js [hunk 2]
```diff
@@ -39,18 +42,49 @@ VmIsStarting.propTypes = {
...
+      <div className="vm-consoles-no-rdp-content">
+        <span>
+          This is a Windows virtual machine but no Service for the RDP (Remote Desktop Protocol) can be found.
+        </span>
+        <br />
+        <span>
+          For better experience accessing Windows console, it is recommended to use the RDP. To do so, create a service:
+          <ul>
+            <li>
+              exposing the <b>3389/tcp</b> port of the virtual machine
```
Comment (rawagner) [inline src/components/VmConsoles/VmConsoles.js]: You already have RDP constant in  https://github.com/kubevirt/web-ui/pull/129 . I propose to move it to web-ui-components and reuse it in this text.

File: src/components/VmConsoles/VmConsoles.js [hunk 3]
```diff
@@ -39,18 +42,49 @@ VmIsStarting.propTypes = {
...
+            </li>
+            <li>
+              using selector: <b>vm.cnv.io/name: {vm.metadata.name}</b>
```
Comment (rawagner) [inline src/components/VmConsoles/VmConsoles.js]: use constant `TEMPLATE_VM_NAME_LABEL`

File: src/components/VmConsoles/VmConsoles.js [hunk 4]
```diff
@@ -39,18 +42,49 @@ VmIsStarting.propTypes = {
...
+            </li>
+            <li>
+              Example: virtctl expose virtualmachine {vm.metadata.name} --name {vm.metadata.name}
+              -rdp --port [UNIQUE_PORT] --target-port 3389 --type NodePort
+            </li>
+          </ul>
+          Make sure, the VM object has <b>spec.template.metadata.labels</b> set to{' '}
+          <b>vm.cnv.io/name: {vm.metadata.name}</b>
```
Comment (rawagner) [inline src/components/VmConsoles/VmConsoles.js]: use constant `TEMPLATE_VM_NAME_LABEL`

Comment (mareklibra) [reply src/components/VmConsoles/VmConsoles.js]: done

File: src/components/VmConsoles/VmConsoles.js (hunk 3 above)
Comment (mareklibra) [reply src/components/VmConsoles/VmConsoles.js]: done

File: src/components/VmConsoles/VmConsoles.js (hunk 2 above)
Comment (mareklibra) [reply src/components/VmConsoles/VmConsoles.js]: done

File: sass/components/_VmConsoles.scss (hunk 1 above)
Comment (mareklibra) [reply sass/components/_VmConsoles.scss]: done

**Pull request merged**